README.md

```diff
@@ -12,13 +12,11 @@ Core is written in Python3 and Django 3.0.3.
 
 #### Ubuntu 18.04 Users
 
-Please install and set up the following packages first. Upgrade if you find the package is already installed:
+Please install these packages and set up your environment in the order listed below. Run an upgrade or update if you find that the package is already installed:
 
 - Python3. Run the 'python3 -V' command to see the version you have installed.
 
-- [PostgreSQL](https://www.postgresql.org/) (Make sure to note Database name, Database Username and Password. Ensure the server is running).
-
-- It is advisable to install Django in a virtual environment. The README uses [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html#basic-installation) to create this environment. You could use any virtualenv package of your choice or install this wrapper with:
+- It is advisable to install Django and the other packages required for development in a virtual environment so this needs to be installed and set up. The README uses [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html#basic-installation) to create this environment. You could use any virtualenv package of your choice or install this wrapper with:
```

Comment:
Having this text `It is advisable to install Django and the other packages required for development in a virtual environment so this needs to be installed and set up.` still reads like you should install Dango first. this will be confusing for a newbie who does not even know Django at all. Please rephrase this whole line to be:

Create a virtual environment in order to install packages. The README uses [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html#basic-installation) to create this environment. You could use any virtualenv package of your choice or install this wrapper with:

Reply:
Okay, thanks.